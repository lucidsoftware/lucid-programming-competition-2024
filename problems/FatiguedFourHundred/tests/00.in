version https://git-lfs.github.com/spec/v1
oid sha256:3067a68b2d1b54fa748b8be23b94aa62880ffbf20a0db0224dead2f6dfd3d373
size 122
