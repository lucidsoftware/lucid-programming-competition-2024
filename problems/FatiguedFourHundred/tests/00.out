85 90 95 100 265.6
80 95 95 100 297.1
85 90 95 100 240.4
85 90 95 100 247.1
85 90 95 100 209.4
